### ISSUE TITLE
make it supported intel atom processor devices!

### ISSUE THREAD

**Issue by NobiZon** [zhuowei/MCPELauncher#182]
Mr.Zhuowei Why you don't make blocklauncher support intel atom processor devices? I know My Zenfone 5 use atom processor. but blocklauncher doesn't support atom devices! many people use zenfone 5 like me. if you make it support atom devices. many people can use it to install mods on it atom devices. So Next update Can you make it support on atom devices? i'll wait until you launched new version that support atom devices!

**Comment by jerbear64** [comment]
On my MeMO Pad (which is x86) BlockLauncher starts fine, but won't apply mods or shaders from texture packs. If the shader pack is in a modded APK, BlockLauncher still won't show it but the Minecraft  app itself will.

**Comment by Parju** [comment]
yeah zhuowei many people waiting update for intel x86 but you doesnt respon to make update for intel 
why ?

**Comment by jeremyrock** [comment]
When does it support Intel atom (Motorola Razr i)

**Comment by Parju** [comment]
@zhuowei blocklauncher have done some updates but why did not resolve the problem for intel first and it looks like you do not care about the problems intel processor
sorry if my words offend and long

**Comment by zhuowei** [comment]
Try http://tinyw.in/bl

**Comment by Parju** [comment]
@zhuowei nice but can't instal mod on zenfone 5

**Comment by jeremyrock** [comment]
Thanks

**Comment by zhizhen99** [comment]
Mr zhuowei now shader or texture pack support for Intel atom???

**Comment by jimchen5209** [comment]
It works well with my Zenfone 6.
Thanks a lot!!!

**Comment by jimchen5209** [comment]
But addons with .apk mods doesn't work.
![screenshot_2015-04-01-19-29-57](https://cloud.githubusercontent.com/assets/10269287/6940172/bb70b36a-d8a6-11e4-8d0c-50b9b30721af.png)
![screenshot_2015-04-01-19-30-02](https://cloud.githubusercontent.com/assets/10269287/6940182/c63ee26c-d8a6-11e4-8ba7-52e8d6715b46.png)
![screenshot_2015-04-01-19-30-12](https://cloud.githubusercontent.com/assets/10269287/6940187/cf84e718-d8a6-11e4-95c5-88ba329a43f7.png)

**Comment by artus9033** [comment]
It crashes on Lenovo s8-50l (intel atom z3745, android KitKat) :-( Plz fix...

**Comment by Lucas3ks** [comment]
Does not work on asus Fonepad 7 with intel atom processor, please correct it.

**Comment by RandomAltThing** [comment]
STOP FUCKING CALLING HIM MR ZHUOWEI

**Comment by jeremyrock** [comment]
It does not work in my Motorola razr i with Intel atom processor, pls fix it.

**Comment by connerfick** [comment]
@RandomAltThing, stop cursing.

**Comment by RandomAltThing** [comment]
@connerfick Don't tell me what I can and can't do, only zhuowei can do that as he owns this repository.

**Comment by connerfick** [comment]
I was asking you to stop not telling you to stop.

**Comment by RandomAltThing** [comment]
@connorfick To quote what you said:

*@RandomaltThing, stop cursing.*

**Comment by OniOniOn-** [comment]
Make new issue if you're having problem. Don't post them on other people's issues.

**Comment by LimEP1227** [comment]
Please fix this

**Comment by zhuowei** [comment]
Please try latest beta: http://tinyw.in/bl

**Comment by yoshikirby43** [comment]
Huh when I start blocklauncher this comes up unfortunately this version of minecraft pe is not compatible with blocklauncher please install minecraft pe for Intel processors. Minecraft pe x86 edition?

**Comment by boby400** [comment]
Please fix this problem :/

**Comment by sliceofcode** [comment]
@yoshikirby43 it means you are not running the "atom" version of blocklauncher, redownload it from the play store.
@boby400 it's being worked on. be patient

**Comment by yoshikirby43** [comment]
No addons support :(

**Comment by jimchen5209** [comment]
Addons needs to be supported itself.
If the addon isn't be supported by author, it still doesn't work.
So... contact the mod's author.
2016年1月18日 上午6:35於 "yoshikirby43" <notifications@github.com>寫道：

**Comment by Kaiquee** [comment]
Sr. zhuowei porque você não se importa com os aparelhos com intel atom? :\

**Comment by skyzelda** [comment]
i have phone Advan X7 intel atom but i am play blocklauncher is not working please update for me

**Comment by dinisgamer** [comment]
I have a acer iconia one 7 b1-730hd and in the newer version (1.12.2) say : block launcher is not support intel devices . download minecraft pé for intel devices

**Comment by dinisgamer** [comment]
_
![screenshot_2016-03-13-14-58-12](https://cloud.githubusercontent.com/assets/17811135/13729393/08fa5244-e92c-11e5-90b1-f28141ba0bfd.png)
_

**Comment by yoshikirby43** [comment]
Did you buy pocket edition or just download an apk off the internet

**Comment by dinisgamer** [comment]
I try all the versions 0.14 and i reboot the device. After that i open the mcpe launcher and the same message appears

**Comment by dinisgamer** [comment]
I buy from aptoide

**Comment by jerbear64** [comment]
"buy" 

"aptoide"

Nope.

**Comment by yoshikirby43** [comment]
Nope you have to buy the full minecraft on Google play

**Comment by Jasper1224** [comment]
ok,new here,have been using this for a while before on a old samsung,recently got a zenfone 2,so it gives the same error as above,and yes,i admit,i was using the online download,not from aptiode,just found it online,but i have bought the app before on amazon,which leads me to my question:does it have to be from the play store,or can it be from amazon?

**Comment by jerbear64** [comment]
Apparently it HAS to be Play Store.

My copy's from Amazon as well and it errors out.

**Comment by Jasper1224** [comment]
well,that's annoying.

**Comment by Jasper1224** [comment]
Also,does the latest beta here support the 0.14.0 build 7 update?

**Comment by dinisgamer** [comment]
In aptoide you can buy 15.0 for free

**Comment by skyzelda** [comment]
My blocklauncher is not working to my phone advan X7 please fix blocklauncher to intel atom

**Comment by zedman2000** [comment]
I run this on an Atom without issue.  I also run this on an i7 without issue.  You can't use addons (APK mods) from authors that have NOT created an Intel version.  Mods (.js / modpack) should without issue.  Same with texture packs.  I use mods and texture packs without issue.

I think as was pointed out earlier, I am also using a REAL copy from Google Play.  It seems to me that most on here (I'm not saying everyone) is not getting an official copy of Minecraft or BlockLauncher, which I believe that almost ALL if not ALL of those "unofficial" copies are for Android running ARM.  So, it does not work.  Please use a REAL copy, and if you have "bought" it for free, you are not using a legal copy.  Minecraft costs money.  They do NOT give it away.  If you own a full version and paid nothing for it, I would wager you got a stolen copy of the game.  Trust Google Play.  I actually think that is the only place to get a legal X86 version of Minecraft - the same as Block Launcher itself.  I'm using all legally purchased copies and have no issue with anything that is listed in this thread.

I do feel for @Jasper1224 though, however Amazon uses ARM in their tablets so I am not surprised it did not work on x86.  That really is not cool.

**Comment by opkiler22789** [comment]
![screenshot_2016-03-26-08-34-25](https://cloud.githubusercontent.com/assets/17113289/14058565/95b901d4-f32d-11e5-99ec-eae09ba30818.png)
так же решыть

**Comment by zahran123** [comment]
![screenshot_2016-06-29-01-48-02 1](https://cloud.githubusercontent.com/assets/20200187/16441778/cde0b402-3df7-11e6-9641-fe691515d35b.jpg)
PLEASE FIX IT ASUS ZENFONE 5 (ASUS T00F)

**Comment by jimchen5209** [comment]
Which version of mcpe do you have?
It says you didn't install the correct version of mcpe.
Make sure you installed mcpe which supports the intel processor.

**Comment by zahran123** [comment]
1.4.0

**Comment by jimchen5209** [comment]
Then find the later build of mcpe 0.14.0,hope this helps you.
(Block Launcher supports 0.14.2 so far)
2016年6月29日 13:59於 "zahran123" <notifications@github.com>寫道：

**Comment by zahran123** [comment]
oh thanks
